main.go

```diff
@@ -186,10 +186,10 @@ func runTBCSim(equip tbc.Equipment, opt tbc.Options, seconds int, numSims int, c
 	stats := tbc.CalculateTotalStats(opt, equip)
 
 	spellOrders := [][]string{
-		{"CL6", "LB12", "LB12", "LB12"},
-		{"CL6", "LB12", "LB12", "LB12", "LB12"},
-		{"CL6", "LB12", "LB12", "LB12", "LB12", "LB12"},
-		{"pri", "CL6", "LB12"}, // cast CL whenever off CD, otherwise LB
+		// {"CL6", "LB12", "LB12", "LB12"},
+		// {"CL6", "LB12", "LB12", "LB12", "LB12"},
+		// {"CL6", "LB12", "LB12", "LB12", "LB12", "LB12"},
+		// {"pri", "CL6", "LB12"}, // cast CL whenever off CD, otherwise LB
```

Comment:
These changes look like something you did while playing around with the sim. Not very consequential, but not part of what you're trying to do.